### PR TITLE
Update README.md to include the AUR option.

### PR DESCRIPTION
**Pull request by RafaelJust**

### CODE REVIEW

**Comment by RafaelJust** [comment]
Whoops i forgot to add a description for this (sorry im kinda new to github)
I made an AUR package for ani-skip here: https://aur.archlinux.org/packages/ani-skip-git
This pull request is just a simple change to the install instructions to reflect that

**Comment by synacktraa** [comment]
Looks good, will merge it soon.

File: README.md [hunk 1]
```diff
@@ -94,6 +94,7 @@ ani-skip -h
 ## Install
 
 - Linux
+  > For Arch linux, ani-skip is available int the AUR as [ani-skip-git](https://aur.archlinux.org/packages/ani-skip-git).
```
Comment (synacktraa) [inline README.md]: Correct the typo please. int -> int

**Comment by RafaelJust** [comment]
I have fixed it now :)

**Pull request merged**